### PR TITLE
Fix 1008994 2

### PR DESCRIPTION
**Pull request by juliusmilan**
related to bugzillas:

https://bugzilla.redhat.com/show_bug.cgi?id=1008994
https://bugzilla.redhat.com/show_bug.cgi?id=1008977

### CODE REVIEW

File: src/plugins/reporter-bugzilla.c [hunk 1]
```diff
@@ -351,9 +354,15 @@ int main(int argc, char **argv)
         }
     }
 
+    char user_bugzilla_conf_path[PATH_MAX];
+    strcpy(user_bugzilla_conf_path, getenv("HOME"));
+    strcat(user_bugzilla_conf_path, USER_HOME_CONFIG_PATH"/bugzilla.conf");
+
     {
-        if (!conf_file)
+        if (!conf_file) {
```
Comment (mhabrnal) [inline src/plugins/reporter-bugzilla.c]: Please, follow the code style.

**Comment by mhabrnal** [comment]
Could you please add some tests as well?

File: src/plugins/reporter-bugzilla.c (hunk 1 above)
Comment (juliusmilan) [reply src/plugins/reporter-bugzilla.c]: done.

**Comment by juliusmilan** [comment]
updated, tests added https://github.com/abrt/abrt/pull/1273

File: src/plugins/reporter-bugzilla.c [hunk 2]
```diff
@@ -351,9 +354,16 @@ int main(int argc, char **argv)
         }
     }
 
+    char user_bugzilla_conf_path[PATH_MAX];
+    strcpy(user_bugzilla_conf_path, getenv("HOME"));
+    strcat(user_bugzilla_conf_path, USER_HOME_CONFIG_PATH"/bugzilla.conf");
+
```
Comment (mhabrnal) [inline src/plugins/reporter-bugzilla.c]: I would rather use `xasprintf` funciton instead.

**Comment by juliusmilan** [comment]
@mhabrnal updated

**Pull request merged**